flancy.psm1

```diff
@@ -156,6 +156,60 @@ function New-Flancy {
         [Parameter(Position=0)]
         [string] $url='http://localhost:8000',
         [Parameter(Mandatory=$false)]
+        [ValidateScript({
+            $PropSet = 'path', 'method', 'script'

```

Comment:
This is good for now, but we're going to have optional ones too that we may need to validate.  For example, type='static' or authrequired=$true.  Just food for thought - won't hold up pull request.  It may also be good to put the validator scriptblock outside of the function, but not sure.  Again, nothing you have to do - just capturing my thoughts.